lib/programmemory.cpp

```diff
@@ -274,6 +274,8 @@ static bool isBasicForLoop(const Token* tok)
 void programMemoryParseCondition(ProgramMemory& pm, const Token* tok, const Token* endTok, const Settings* settings, bool then)
 {
     auto eval = [&](const Token* t) -> std::vector<MathLib::bigint> {
+        if (!t)
+            return std::vector<MathLib::bigint>{};
```

Comment:
`return {};`?

Reply:
Possibly. But I just copied from a few lines down.

Reply:
Seems like we don't have the corresponding clang-tidy check enabled.